### ISSUE TITLE
Tutorial missing (apparently) important detail about including in module

### ISSUE THREAD

**Issue by beezee** [egonSchiele/contracts.ruby#96]
I wrote a gem to define some Contracts for a few nested types, and included in the module defined by my gem.

My tests passed, but when I bundled it into a rails project rails would crash on start with the following error:

```
eigenclass.rb:13:in `lift': private method `singleton_class?' called
```

Turns out that including Contracts::Modules *before* Contracts fixes this problem, but the tutorial actually suggests the opposite order, which is what seems to have caused the error in the first place.

**Comment by beezee** [comment]
Nope, confused myself and jumped the gun. I am not finished debugging this problem yet. Closing for now but if anyone recognizes the issue would love to hear.

**Comment by beezee** [comment]
So it appears that activesupport defines the singleton_class? method on Class with private visibility.

It also appears that they are taking that out - https://github.com/rails/docrails/commit/dc6efd387faf7f11e93877cb6c74033d0b73e75f

But I've upgraded to rails 4.2.0 and I still have that code in my local gemset, so I'm guessing this is not released into stable on rubygems yet.

In the meantime, I'll fork and open a PR to use send from Eigenclass.lift and drop the explicit self receiver from the call in MethodDecoractors.decorate. Hopefully this is not too objectionable.

**Comment by waterlink** [comment]
Fixed by #97
